README.md

```diff
@@ -292,17 +292,16 @@ this:
 ```scheme
 ; Configuration
 (#language! rust)
-(#indent-level! 4)
+(#indent! "    ")
```

Comment:
Because of how common using multiple spaces is, and how unreadable a bunch of spaces between double-quotes is, I would avise keeping `(#indent-level! <num>)` as an alternative to `(#indent! <string>)`.

(maybe use `#indent! "    "` in one of the query file in order to make sure that the code is used)

Reply:
Hej!

There are a few reasons that I hesitate to do that.

1. I want to keep this code as simple as possible because it's quite hacky as is.
2. We are moving to a toml based configuration file before we move to 0.2, which could possibly* allow both integers and strings in a single field.

* Not 100% about this, TBD

Reply:
Maybe, even then, I think this approach is good enough for people to get started with other forms of indentation, without putting too much effort into it. Effort that is likely to be wasted when moving to a configuration file.